turbo.json

```diff
@@ -1,7 +1,10 @@
 {
   "$schema": "https://turborepo.org/schema.json",
   "pipeline": {
-    "prebuild": {},
+    "preinstall": {},
+    "prebuild": {
+      "dependsOn": ["^prebuild", "preinstall"]
+    },
     "build": {
       "dependsOn": ["^build", "prebuild"],
```

Comment:
we should add preinstall here?

Reply:
yes, some prebuild commands require forc-bin to be ready already